src/input/mpegts/satip/satip.c

```diff
@@ -384,12 +398,6 @@ satip_device_hack( satip_device_t *sd )
     sd->sd_pids_max    = 128;
     sd->sd_pids_len    = 2048;
     sd->sd_no_univ_lnb = 1;
-  } else if (strstr(sd->sd_info.manufacturer, "AVM Berlin") &&
```

Comment:
I'm not sure what you mean here, either. If I leave those lines there tvheadend will always apply those settings and ignore user-set preferences. I wanted them optional just in case AVM ever releases a firmware update, and thus I moved the check earlier so that there are sane defaults.